Doc/library/os.rst

```diff
@@ -1659,6 +1659,27 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
       :exc:`InterruptedError` exception (see :pep:`475` for the rationale).
 
 
+.. function:: readinto(fd, buffer, /)
+
+   Read from a file descriptor *fd* into a mutable
+   :ref:`buffer protocol <bufferobjects>` *buffer*.
+
+   The *buffer* should be mutable and :term:`bytes-like <bytes-like object>`. On
+   success, returns the number of bytes read. Less bytes may be read than the
```

Comment:
English question but do we say "Less bytes may be" or "Fewer bytes may be"

Reply:
`getrandom` at least says "less bytes than requested", and `copy_file_range` says "less than the
   amount requested".  No instances of `fewer` in `os.rst` currently, so I think "less" is more the current documentation norm